src/pharmpy/statements.py

```diff
@@ -593,6 +590,38 @@ def set_lag_time(self, compartment, lag_time):
         return new_comp
 
 
+def _is_positive(expr: sympy.Expr) -> bool:
+    return sympy.ask(
+        sympy.Q.positive(expr), _assume_all(sympy.Q.positive, _free_images_and_symbols(expr))
+    )
+
+
+def _free_images_and_symbols(expr: sympy.Expr) -> Set[sympy.Expr]:
```

Comment:
Not entierly clear what `images` is referring to.

Reply:
Not sure it is the best name. It is supposed to mean `image` as in https://en.wikipedia.org/wiki/Image_(mathematics).

Reply:
What sympy terms `Function`?

Reply:
I think `Function` would be `f`. Here an `image` is `f(x)`.

Reply:
Ok, sympy calls this `Application`, but that is even less helpful. If the functions are merged or moved into the caller the names could be kept.